MAINTAINERS.md

```diff
@@ -0,0 +1,2 @@
+* Augustin Husson <husson.augustin@gmail.com> [@nexucis](https://github.com/Nexucis)
+* Julius Volz <julius.volz@gmail.com> [@juliusv](https://github.com/juliusv)
```

Comment:
ah maybe since it's a promlabs repository would you prefer to use your promlabs email @juliusv  ?

Reply:
Hmm, good question. Yeah, here julius.volz@promlabs.com might be better :)